packages/opc-back-to-top/README.md

```diff
@@ -0,0 +1,127 @@
+# This is opc-back-to-top 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/
+
+A standardized web component based on Lit element for Red Hat One Platform back to top.
+
+## Install
+
+```sh
+npm install
+```
+
+## Usage
+plain back to top
+```html
+<opc-back-to-top></opc-back-to-top>
```

Comment:
@anjneeksharma this Readme doesn't follow the standard Readme format that we have created. Please update it accrodingly

Reply:
all the attributes and slots should be mentioned separately

Reply:
@diwanshi i am not able to understand your this comment, can you please elaborate bit ?

Reply:
@anjneeksharma can u use the standard readme format that we are using in opc-header component